src/components/AudioPlayer/index.js

```diff
@@ -294,18 +303,26 @@ class AudioPlayer extends Component {
                 updatePlayed={this.updatePlayed}
                 updatePlaying={this.updatePlaying}
                 updatePlayable={this.updatePlayable}
+                updatePrevProgress={this.updatePrevProgress}
                 width={width}
                 autoplay={autoplay}
                 editor={editor}
                 endSong={this.endSong}
                 _fonts={_fonts}
+                active={active}
+                topScreen={topScreen}
+                backgroundAudio={
+                  typeof backgroundAudio !== 'undefined'
+                    ? backgroundAudio
+                    : true
```

Comment:
Note: The audio player works on web, iOS, and Android, and supports background play.
First, make sure you've installed the audio player component from the Adalo Marketplace, then click the plus button in the left panel, find the audio player component and drag it on to a screen in your app.
You can adjust the following in the left panel:

Reply:
If you added default props then backgroundAudio won't ever be undefined, so you're all good to go. But doesn't hurt to leave this logic in.